mewarpx/mewarpx/util.py

```diff
@@ -46,3 +56,16 @@ def mkdir_p(path):
             pass
         else:
             raise
+
+
+def ideal_gas_density(p, T):
+    """Calculate neutral gas density from the ideal gas law.
```

Comment:
I would add the Torr and cm^-3 units to this summary docstring so people are most likely to see it.